io/io/inc/ROOT/RIoUring.hxx

```diff
@@ -0,0 +1,63 @@
+/*************************************************************************
+ * Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_RIoUring
+#define ROOT_RIoUring
+
+#include <cstring>
+#include <liburing.h>
+#include <liburing/io_uring.h>
+
+#include <ROOT/RError.hxx>
+#include <ROOT/RLogger.hxx>
```

Comment:
Ah, I think the problem is that RError and RLogger are ROOT7 facilities.

Reply:
Let's throw an `std::runtime_error` and use the facilities from `TError` for the time being.

Reply:
I think the idea here is that any ROOT IO class (`TTree` etc) could use `RIoUring` if they choose.

Reply:
Thanks!